main.js

```diff
@@ -47,11 +48,21 @@ define(function (require, exports, module) {
     nodeDomain.exec('lintFile', fullPath, projectRoot)
       .then(function (report) {
         if (report.results.length > 1) {
-          console.warn('ESLint returned multiple results, where only one set was expected');
+          //console.warn('ESLint returned multiple results, where only one set was expected');
         }
         var results = report.results[0];
-        var remapped = remapResults(results.messages);
-        deferred.resolve(remapped);
+
+        nodeDomain.exec('getESLintVersion').then(function(res) {
```

Comment:
This is wrong - you shouldn't call another method here again, just return the version in the `lintFile` results. Attach it to the `report` object.